File: WebApplication1/Content/components/Crosswords/Models/LeaderboardItemViewModel.ts
export class LeaderboardItemViewModel {
    //isCoOp: boolean;
    millisecondsElapsed: number;
    //usedHints: boolean;
    userId: string;
    userName: string;
    isSolved: boolean;

    constructor() {
        //this.isCoOp = false;
        this.millisecondsElapsed = 0;
        //this.usedHints = false;
        this.userId = "";
        this.userName = "";
        this.isSolved = false;
    }
}

File: WebApplication1/ViewModels/Crosswords/LeaderboardItemViewModel.cs
namespace CrossWorldApp.ViewModels.Crosswords;

public class LeaderboardItemViewModel
{
    public bool IsCoOp { get; set; }
    public string UserName { get; set; }
    public double MillisecondsElapsed { get; set; }
    public bool UsedHints { get; set; }
    public string UserId { get; set; }
    public bool IsSolved { get; set; }

    public LeaderboardItemViewModel(Models.Solve solve)
    {
        IsCoOp = solve.IsCoOp;
        UserName = solve.UserName;
        MillisecondsElapsed = solve.MillisecondsElapsed;
        UsedHints = solve.UsedHints;
        UserId = solve.UserId;
        IsSolved = solve.IsSolved;
    }
}